primitives/src/lib.rs

```diff
@@ -92,6 +92,7 @@ pub type Price = FixedU128;
 pub type Timestamp = u64;
 
 pub const SECONDS_PER_YEAR: Timestamp = 365 * 24 * 60 * 60;
+pub const TOKEN_DECIMALS: u8 = 18; //FixedU128::DIV;
```

Comment:
The source of truth of token decimals is controlled by other chains, we should not maintain it in this chain.

Reply:
Removed